CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Version 0.17.5
 - Improved support of keymodifier for linux, arrow keys, function keys, home keys etc.
 - Add `SetTitle` command to change the terminal title.
+- Mio 0.7 update
```

Comment:
@TimonPost this is not part of `0.17.5` right?

Reply:
I will update the changelog before releasing